docs/tutorial.rst

```diff
@@ -505,21 +508,24 @@ Example::
     from toil.job import Job
 
     class HelloWorld(Job):
-        def __init__(self, message):
+        def __init__(self, message, inputFileID):
             Job.__init__(self,  memory="2G", cores=2, disk="3G")
-            self.message = message
+            self.inputFileID = inputFileID
+
+        with fileStore.readGlobalFileStream(self.inputFileID) as f:
+            with fileStore.writeGlobalFileStream() as (fH, jobStoreFileID):
+                fH.write(f.read() + 'World!')
+            return jobStoreFileID
 
-        def run(self, fileStore):
-            return "Hello, world!, here's a message: %s" % self.message
 
     if __name__=="__main__":
         options = Job.Runner.getDefaultOptions("./toilWorkflowRun")
         options.logLevel = "INFO"
         job = HelloWorld("enter the void empty and become wind")
         with Toil(options) as toil:
-            jobStoreFileID = toil.fileStore(job).importFile('s3://somebucketname/anditskey')
-            toil.exportFile(jobStoreFileID, 'file:///some/local/path')
-            toil.run(job)
+            jobStoreFileID = toil.importFile('file://some/local/path')
```

Comment:
My mistake I forgot about that... but there is code already in master that asserts this in toil.jobStores.fileJobStore.FileJobStore._extractPathFromUrl()